seed/static/seed/js/controllers/organization_settings_controller.js

```diff
@@ -119,10 +123,22 @@ angular.module('BE.seed.controller.organization_settings', []).controller('organ
       value: 2
     }];
 
-    const resize_textarea = function () {
+    $scope.confirm_delete = function (org) {
+      $uibModal.open({
+        templateUrl: urls.static_url + 'seed/partials/delete_org_modal.html',
+        controller: 'delete_org_modal_controller',
+        backdrop: 'static',
+        keyboard: false,
```

Comment:
Great idea 💯  Limiting opportunities to just randomly close the modal at weird/unexpected times.